apps/docs/app/routes/komponenter/accordion.tsx

```diff
@@ -0,0 +1,154 @@
+import { InteractiveCode } from "~/features/interactive-code/InteractiveCode";
+import { LinkableHeading } from "~/features/linkable-heading/LinkableHeading";
+
+type AccordionPageProps = {
+  title: string;
+  children: React.ReactNode;
+};
+
+const AccordionDetails = ({ title, children }: AccordionPageProps) => (
+  <Stack spacing={1} textStyle="sm">
+    <LinkableHeading as="h2" textStyle="xl-display">
+      {title}
+    </LinkableHeading>
+    {children}
+  </Stack>
+);
+
+export default function AccordionsPage() {
+  return (
+    <ComponentDocs
+      title="Accordion"
+      description="Expandables er utvidbare elementer."
+    >
+      <DemoArea />
+      <Guidelines />
+    </ComponentDocs>
+  );
+}
+
+const DemoArea = (props: BoxProps) => {
+  const { currentProps, propList, onPropsChange } = usePlaygroundProps([
+    {
+      name: "variant",
+      values: ["list", "outline", "card"],
+      defaultValue: "card",
+      type: "select",
+    },
+    {
+      name: "size",
+      values: ["sm", "md", "lg"],
+      defaultValue: "md",
+      type: "select",
+    },
+    {
+      name: "title",
+      defaultValue: "Accordion Item #1",
+      type: "input",
+    },
+  ]);
+  const { title, ...remainingProps } = currentProps;
+  const code = `
+  <Accordion
+  allowToggle
+    ${toPropsString(remainingProps)}
+    
+   >  
+   <ExpandableItem title="${title}" icon={<TrainOutline30Icon/>}>
+   Nå kan du teste utvidbare elementer 😎
+   </ExpandableItem>
+  </Accordion>`;
+  return (
+    <Box {...props}>
+      <ComponentPlayground
+        code={code}
+        propList={propList}
+        currentProps={currentProps}
+        onPropsChange={onPropsChange}
+      />
+    </Box>
+  );
+};
+
+const Guidelines = (props: BoxProps) => {
+  return (
+    <Stack {...props} spacing={8}>
+      <Stack spacing={3}>
+        <LinkableHeading as="h2" textStyle="xl-display">
+          Retningslinjer
+        </LinkableHeading>
+        <Text>
+          Expandables eller utvidbare elementer skal brukes der det er behov for
+          å skjule informasjon eller grafiske elementer, for å holde designet
+          ryddig, gjøre det enklere å finne frem, sette et tydelig hierarki for
+          informasjonsflyten. Dette kan være alt fra kun tekst (se F.A.Q), til
+          stoppesteder i reisedetaljer eller billettinformasjon. Disse
+          komponenten kan utvides om det er en behov for mer egne elementer.
+          Dette skal brukes som et malverk slik at størrelser, farger, states og
+          interaksjon blir overholdt.
+        </Text>
+        <InteractiveCode>
+          {`
+<Accordion variant="card" size="md" > 
+<Stack spacing={2}>
+  <ExpandableItem title="Accordion #1" headingLevel="h3" icon={<TrainOutline30Icon/>}>
+    <Text>Dette er den første accordions.</Text>
+  </ExpandableItem>
+  <ExpandableItem title="Accordion #2" headingLevel="h3">
+    <Text>Dette er den andre accordions</Text>
+  </ExpandableItem>
+  </Stack>
+</Accordion>
```

Comment:
Samme som over - formatteringen her må være on fleek 😁 

```suggestion
<Accordion variant="card" size="md"> 
  <Stack spacing={2}>
    <ExpandableItem title="Første panel" headingLevel="h3" icon={<TrainOutline30Icon/>}>
      <Text>Dette er det første panelet.</Text>
    </ExpandableItem>
    <ExpandableItem title="Andre panel" headingLevel="h3">
      <Text>Dette er det andre panelet</Text>
    </ExpandableItem>
  </Stack>
</Accordion>
```